### DISCUSSION TITLE
Opening mailto: links sandboxed Brave?

### DISCUSSION THREAD

**Discussion by e-t-l** [sandboxie-plus/Sandboxie#3951] [Q&A Feedback] [answered]
I have Brave (sandboxed, but installed on the host system) set as the system's default app to open `mailto:` links. In Brave, Gmail is set as the mail handler. /If I hit <kbd>Win</kbd>+<kbd>R</kbd> and enter "mailto:test@example.com", it launches Brave appropriately. If I try to open a mailto address in any other app, including Word (unsandboxed) or even Firefox (in the same sandbox as Brave), **nothing happens.** (In Firefox, the default mail handler is set to Brave.exe.)

Before I create a whole bug report issue about this, is this normal/replicable behavior? Am I missing something to make Brave open `mailto:` links consistently?

**Comment by offhub** [comment, marked as answer]
I tried it on a Windows 11 23H2 virtual machine with default settings.

1. After installing the Brave browser, I set it as the default MAILTO application through the system.
2. I assigned the browser as a forced application to run in the sandbox.
3. After running it outside the sandbox and logging in to the Gmail account, I set it as the default email handler.
4.
    - Ctrl+R
    - WordPad [Unsandboxed, Sandboxed (same/different) sandbox]
    - Firefox [Unsandboxed, Sandboxed (same/different) sandbox]
    - MS Edge [Unsandboxed, Sandboxed (same/different) sandbox]

5. In all the above cases, the sandboxed Brave browser opened and redirected to the Gmail compose page.

I recommend that you try this with a new and empty sandbox.

**Reply by e-t-l** [reply]
I appreciate you taking the time to troubleshoot this. I'll try making a fresh box.
